Demo/Demo/ViewModels/BaseViewModel.swift

```diff
@@ -257,6 +296,13 @@ class BaseViewModel: ObservableObject, PayPalWebCheckoutDelegate, CardDelegate {
         return CoreConfig(clientID: DemoSettings.clientID, accessToken: token, environment: DemoSettings.environment.paypalSDKEnvironment)
     }
 
+    func getNativeCheckoutClient() async throws -> PayPalClient {
+        guard let config = await getCoreConfig() else {
+            throw CoreSDKError(code: 0, domain: "Error initializing paypal webcheckout client", errorDescription: nil)
```

Comment:
Same error comment here, we can instead bubble up an error when we attempt to fetch the config.